include/d/col/bg/d_bg_w_base.h

```diff
@@ -183,6 +183,10 @@ class dBgW_Base : public cBgW_BgId {
         field_0x24 |= 2;
     }
 
```

Comment:
This is just a flag kind of thing, unless there was a reason that it was named stick, the flag is unknown. The other one was for like if arrow got stuck in collision iirc